src/main/java/online/book/store/repository/book/provider/TitleSpecificationProvider.java

```diff
@@ -0,0 +1,22 @@
+package online.book.store.repository.book.provider;
+
+import online.book.store.model.Book;
+import online.book.store.repository.book.spec.BookSpecificationBuilder;
+import online.book.store.repository.specification.SpecificationProvider;
+import org.springframework.data.jpa.domain.Specification;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TitleSpecificationProvider implements SpecificationProvider<Book> {
+
+    @Override
+    public String getKey() {
+        return BookSpecificationBuilder.TITLE;
```

Comment:
I'd rather do other way around so that this constant would belong to this class and imported it to the `Builder`